### ISSUE TITLE
Possible extra aria-labelledby in Keyboard Shortcuts dialog

### ISSUE THREAD

**Issue by terracoda** [phetsims/scenery-phet#494]
From https://github.com/phetsims/gravity-force-lab-basics/issues/118#issuecomment-488117763
@zepumph, I think there might be an extra ` aria-labelledby` because I am hearing an end of group announcement after the starting heading block of the dialog. It sounds odd to me to hear the end of the group when I am near the top of the dialog.

Note this is not new, I have heard this "end group" announcement before, and I just looked in  the code in Gravity Force Lab Basics dialog to see why. Here's a screen shot:
<img width="938" alt="Screen Shot 2019-04-30 at 10 54 35 PM" src="https://user-images.githubusercontent.com/1902116/56993534-0043ed80-6b9d-11e9-98c7-01dd95bf0807.png">

It is possible we do not need the second `aria-labelledby` attribute.

**Comment by zepumph** [comment]
I didn't realize that having an aria-labelledby attribute for the title div would effect how it sounded with the grouping. I removed the extra aria-labelledby and now there is just a single one on the parent. @terracoda please review that piece.


Furthermore, while testing this, I found that the dialog doesn't sound that good when opening with NVDA (chrome/firefox). Here is what I get when I open the dialog read to me (in text form):
```
Keyboard Shortcuts  dialog
Close
button
Keyboard Shortcuts
button
Close
heading  level 1
Keyboard Shortcuts
Tab to get started.
heading  level 2
‪Move Spheres‬ . . . 
``

I don't understand why the close button is repeated, nor why Keyboard Shortcuts gets said thrice! I perhaps would only expect it twice (once as the dialog label, and once as the heading. @terracoda do you get this at all?

**Comment by terracoda** [comment]
@zepumph, I just tested and it sounded just about perfect in VoiceOver.
Upon opening, I got the following automatically:
> Keyboard Shortcuts, web dialog, with 3 items, Close button

Then when I used the arrows keys to read (down arrow), I got the content in order, starting from the H1, and no repetition.

Quite a while ago, when @jessegreenberg and were getting ready to launch BASE I think, we worked on getting the opening message to sound as good as we could. 

We should make sure we haven't made code updates that reduce the quality of the experience (too much) across screen readers, but this dialog is really tricky.

FYI, the close button is first because it is the only focusable item. If it is placed last, users tend to read the dialog backwards, or get confused by its placement.

Ideally, on opening something reasonable and brief should be read out automatically, something like:
- The name of the dialog
- How many items or headings there are
- and possible where focus is currently

This is exactly what VoiceOver is currently doing, though the item count is not correct. I think VoiceOver is just counting the three items (Close Button, H1 heading, Tab to get start paragraph) that are that header-like block.

I'm not sure why NVDA is reading out things twice or thrice. We may not need `aria-labelledby` at all, but I am not 100% sure.

@zepumph, what exactly is read out automatically upon opening the dialog. I think that's most important.

**Comment by terracoda** [comment]
It's possible more semantic elements, like `header` instead of `div` could help, but I wouldn't worry too much about that right now. The main things are:
1. ensuring users can't get back into the sim without closing the dialog, and
2. a decent or "best we can get" opening message on as many screen readers as possible.

Item 1 takes precedence over item 2 and that right now is still working fine.

**Comment by jessegreenberg** [comment]
Here is the issue from BASE discussing how the dialog is read by NVDA and VO https://github.com/phetsims/balloons-and-static-electricity/issues/399. It sounds like things are sounding similar to the way they were left in that issue, see specifically https://github.com/phetsims/balloons-and-static-electricity/issues/399#issuecomment-382460634 and https://github.com/phetsims/balloons-and-static-electricity/issues/399#issuecomment-382457609

**Comment by zepumph** [comment]
Alright! I'm glad everyone is on the same page. I don't think it sounds bad per-say, I just was curious about how it sounds in NVDA. It is less strange now that I realize the priorities, mainly getting it to sound good when opened.
